src/components/core/base/test/VisualizationLoadingHOC.spec.tsx

```diff
@@ -403,4 +408,44 @@ describe('VisualizationLoadingHOC', () => {
             expect(nullRes).toBe(null);
         });
     });
+
+    describe('Exporter', () => {
+        beforeAll(() => {
+            sdk.clone = jest.fn(() => sdk);
+            sdk.report.exportResult = jest.fn((_, __, exportConfig: IExportConfig) => {
+                if (exportConfig.format === 'xlsx') {
+                    return Promise.resolve({ uri: '/bar' });
+                }
+                return Promise.reject(new Error('Error!!!'));
+            });
+        });
+
+        afterAll(() => {
+            jest.resetAllMocks();
+        });
+
+        it('should return URI when call exportResult successfully', async () => {
+            const onExportReady = (exportResult: IExportFunction) => {
+                exportResult({ format: 'xlsx' })
+                    .then((result: IExportResponse) => { expect(result.uri).toEqual('/bar'); });
+            };
+            const wrapper = await createComponent({
```

Comment:
these tests are badly rewritten to await, are passing all the time even with wrong expects. Await should be where `testUtils.delay.....then(`
was used and this `testUtils.delay` should still be here

Reply:
createComponent is not async function